### ISSUE TITLE
ROOT 6.24/06, aarch64: SOFIE related symbol not found at startup

### ISSUE THREAD

**Issue by gganis** [root-project/root#10877]
Could not find any duplicate.

### Describe the bug
When starting ROOT on an arm64 machine a symbol-not-found message appears for about 10 times before getting to the prompt.
The message is related to SOFIE:

```
 IncrementalExecutor::executeFunction: symbol '_ZN4TMVA12Experimental5SOFIE8INTERNAL22make_ROperator_ReshapeERKN4onnx9NodeProtoERKNS3_10GraphProtoERSt13unordered_mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEENS1_11ETensorTypeESt4hashISG_ESt8equal_toISG_ESaISt4pairIKSG_SH_EEE' unresolved while linking function '_GLOBAL__sub_I_cling_module_0'!
You are probably missing the definition of TMVA::Experimental::SOFIE::INTERNAL::make_ROperator_Reshape(onnx::NodeProto const&, onnx::GraphProto const&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, TMVA::Experimental::SOFIE::ETensorType, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, TMVA::Experimental::SOFIE::ETensorType> > >&)
Maybe you need to load the corresponding shared library?
Symbol found in '/cvmfs/sft.cern.ch/lcg/releases/ROOT/6.26.04-22463/aarch64-centos7-gcc11-opt/lib/libROOTTMVASofieParser.so'; did you mean to load it with '.L /cvmfs/sft.cern.ch/lcg/releases/ROOT/6.26.04-22463/aarch64-centos7-gcc11-opt/lib/libROOTTMVASofieParser.so'?
```

ROOT 6.24/06 works fine.

### Expected behavior
No warning message

### To Reproduce
1. Login on an ARM amchine with cvmfs, e.g. techlab-arm64-thunderx2-01 
2. Source dev4arm
     `source /cvmfs/sft.cern.ch/lcg/views/dev4/Wed/aarch64-centos7-gcc11-opt/setup.sh`
3. Start ROOT, e.g. `root -b -q`

### Setup
1. ROOT version: 6.26/04
2. Operating system: aarch64, centos7
3. LCG stack.

### Additional context
Affecting ATLAS (first noticed by Asoka Da Silva)

**Comment by lmoneta** [comment]
Hi, 
This is a problem with the module/pre-compiled module generation. Is the configure option 
`runtime_cxxmodules` enabled for that build ?

**Comment by andresailer** [comment]
Hi Lorenzo,
runtime_cxxmodules is enabled by default, and we don't disable it, as far as I can tell.
I don't see the option being printed by root-config however, any other way to check this is enabled to be sure?
Thanks,
Andre

**Comment by lmoneta** [comment]
Ok, I will connect to the machine and investigate it. I was asking about runtime_cxxmodules, because I think I have thinking this was a problem by using the pre-compiled headers, which are used when runtime_cxxmodules is disabled.

**Comment by stephanlachnit** [comment]
I have the same problem as well (on ROOT 6.24/04, amd64). I compiled with both `cxxmodules` and `runtime_cxxmodule` disabled.

**Comment by stephanlachnit** [comment]
See https://github.com/ellert/root/pull/1#issuecomment-1318452190, this is fixed in 6.26.08

**Comment by lmoneta** [comment]
This has been fixed in 6.26.08 and in 6.28 the namespace TMVA::Experimental::SOFIE::INTERNAL has been removed in 6.28
